src/mds/MDCache.h

```diff
@@ -568,6 +592,9 @@ class MDCache {
   friend class Locker;
   friend class Migrator;
   friend class MDBalancer;
+  // FIXME hack so that the StrayManager can re-insert inode to inode_map
+  // on abort_queue
+  friend class StrayManager;
```

Comment:
This is from when I was doing the early removal of inode from inode_map, and was hoping to find a way to avoid the call back into mdcache.  I sort of came to terms with StrayManager being more of a sub-class to mdcache than a separate thing, so this is no longer a FIXME - updated.